src/swapper/swapper.py

```diff
@@ -30,10 +33,10 @@ def delete_locks(fc_owner, fc_name, db_owner):
     db_connect = arcpy.ArcSDESQLExecute(db_owner)
 
     sql = dedent(
-        f'''SELECT * FROM sde.SDE_process_information
-        WHERE SDE_ID IN(SELECT SDE_ID FROM sde.SDE_table_locks
-        WHERE registration_id = (SELECT registration_id FROM sde.SDE_table_registry
-        WHERE table_name = '{fc_name}' AND owner = UPPER('{fc_owner}')));
+        f'''SELECT * FROM SDE_process_information
+        WHERE SDE_ID IN(SELECT SDE_ID FROM SDE_table_locks
+        WHERE registration_id = (SELECT registration_id FROM SDE_table_registry
+        WHERE table_name = UPPER('{fc_name}') AND owner = UPPER('{fc_owner}')));
```

Comment:
```suggestion
        WHERE UPPER(table_name) = UPPER('{fc_name}') AND UPPER(owner) = UPPER('{fc_owner}')));
```

Reply:
I've been misunderstanding this part of a SQL query forever. I thought that this would have broken the query because the field names are lower case. I guess I need to think about it in terms of the field name will be substituted with the value.